src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date;
+};
+
+type Props = {
+  name: string;
+};
+
+const getFormattedDate = (value: Date) => value.toUTCString().slice(-12, -4);
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(getFormattedDate(this.state.today));
```

Comment:
Don't use state here, because it can be outdated, save `new Date()` result to constant